ibis/backends/bigquery/__init__.py

```diff
@@ -1222,6 +1297,7 @@ def connect(
     auth_external_data: bool = False,
     auth_cache: str = "default",
     partition_column: str | None = "PARTITIONTIME",
+    generate_job_id_prefix: Callable[[], str | None] | None = None,
```

Comment:
As i noted in https://github.com/ibis-project/ibis/issues/11230#issuecomment-2901159295 (which wasn't commented on), i think this `bigquery/__init__.py::connect()` function is legacy unused.

I'd like @cpcloud 's input, but i think it should be removed to avoid further contributor confusion.

Reply:
Thanks!